### PR TITLE
doc: release note 24.11.0

### PR DESCRIPTION
**Pull request by rahulguptajss**

### CODE REVIEW

**Review by Hardikl — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,34 +4,34 @@
...
-
-
-- Harvest supports Top clients metrics collection
-- Harvest supports recording and replaying HTTP requests
-- Harvest generates fips compliant container image
-- Grafana import allows rewriting cluster label
+- SnapMirror Dashboard now includes relationship details from the source perspective and has been renamed to "ONTAP: SnapMirror Sources".
+- Health Dashboard's emergency events panel now includes all emergency EMS events from the last 24 hours.
+- Network Dashboard
+- Includes Link Aggregation Group (LAG) metrics
```
Comment (cgrinds) [inline CHANGELOG.md:16]: should this and the next bullet be a sub-bullet of Network dashboard?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,34 +4,34 @@
...
-  - Power dashboard includes "Average Power Consumption (kWh) Over Last Hour" panel.
-  - SVM dashboard includes Top SVM by Space panel.
-  - Volume Deep Dive dashboard includes other ops panel.
-
-
-- Harvest supports Top clients metrics collection
-- Harvest supports recording and replaying HTTP requests
-- Harvest generates fips compliant container image
-- Grafana import allows rewriting cluster label
+- SnapMirror Dashboard now includes relationship details from the source perspective and has been renamed to "ONTAP: SnapMirror Sources".
```
Comment (cgrinds) [inline CHANGELOG.md:13]: let's use lowercase dashboard instead of Dashboard throughout

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,34 +4,34 @@
...
+- Health Dashboard's emergency events panel now includes all emergency EMS events from the last 24 hours.
+- Network Dashboard
+- Includes Link Aggregation Group (LAG) metrics
+- Adds Ethernet port details
+- S3 Object Storage dashboard includes panels for s3 metrics for SVM.
```
Comment (cgrinds) [inline CHANGELOG.md:18]: S3 metrics instead of s3 metrics

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,34 +4,34 @@
...
+- Tenant Dashboard
+- Adds Tenant/Bucket Capacity Growth Chart
```
Comment (cgrinds) [inline CHANGELOG.md:20]: sub-bullet of Tenant dashboard

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,34 +4,34 @@
...
+- Includes average size per object details for each bucket
```
Comment (cgrinds) [inline CHANGELOG.md:21]: sub-bullet of Tenant dashboard

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,34 +4,34 @@
...
+- :rocket: Performance Improvements:
+- Reduced memory footprint by optimizing memory allocations when serving metrics.
+- Reduced API calls when using the RestPerf collector.
 
-- :ear_of_rice: Harvest includes new templates to collect:
-
-- :closed_book: Documentation additions
+- Harvest supports Top clients metrics collection. [More details](https://netapp.github.io/harvest/latest/plugins/#volumetopclients).
+- Harvest supports recording and replaying HTTP requests.
+- Harvest now provides a FIPS-compliant container image, available as a separate image.
```
Comment (cgrinds) [inline CHANGELOG.md:33]: as a separate image (`ghcr.io/netapp/harvest:24.08.0-1-fips`)

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,34 +4,34 @@
...
+- Grafana Import allows rewriting the cluster label during import.
```
Comment (cgrinds) [inline CHANGELOG.md:34]: Grafana import or `bin/grafana import`

**Review by Hardikl — APPROVED** [review]

**Pull request merged**